src/javascripts/components/cards/dinoCards.js

```diff
@@ -9,9 +9,16 @@ const authedDinoCardView = (dinoObject) => {
         <div>
           <h3 class="card-text card-header">${dinoObject.name}</h3>
         </div>
-        <button type="button" id="${dinoObject.dinoId}" class="btn btn-info update-dino">Edit</button>
+        <button type="button" id="${dinoObject.dinoId}" class="btn btn-info update-dino card-btns"><i class="fas fa-pen"></i></button>
+        <button type="button" id="${dinoObject.dinoId}" class="btn btn-info delete-dino card-btns"><i class="fas fa-trash-alt"></i></button>
       </div>
     </div>`;
+
+  $('body').on('click', 'button.delete-dino', (e) => {
+    $(`.card#${e.currentTarget.id}`).remove();
```

Comment:
in staying consistent with your other on click events, maybe consider also storing e.currentTarget.id into firebaseKey variable

Reply:
done!